src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/x64/mvn.cpp

```diff
@@ -0,0 +1,276 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_layer_tests/classes/mvn.hpp"
+#include "shared_test_classes/single_layer/mvn.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+#include "test_utils/fusing_test_utils.hpp"
+#include <ngraph_functions/builders.hpp>
+#include <common_test_utils/ov_tensor_utils.hpp>
+
+using namespace InferenceEngine;
+using namespace CPUTestUtils;
+using namespace ngraph::helpers;
+using namespace ov::test;
+
+
+namespace CPULayerTestsDefinitions {
+namespace MVN {
+namespace {
+
+const std::vector<bool> normalizeVariance = {
+       false
+};
+
+std::vector<ElementType> inpPrc = {
+        ElementType::bf16
+};
+std::vector<ElementType> outPrc = {
+        ElementType::bf16
+};
+
+std::vector<CPUSpecificParams> cpuParams_4D = {
+        CPUSpecificParams({nhwc}, {nhwc}, {}, {}),
```

Comment:
It seems unclear why nhwc is not supported by arm / acl
Is it so? If yes, is it by design or we just haven't finished the executor implementation?